lib/user.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'db_connect'
+
+class User
+  attr_reader :id, :handle, :email, :password
+
+  def initialize(id:, handle:, email:, password:)
+    @id = :id
+    @handle = handle
+    @email = email
+    @password = password
+    @logged_in = false
+  end
+
+  def self.create(handle:, password:, email:)
+    connection = DBConnect.check_environment
+    result = connection.exec_params('INSERT INTO users (handle, password, email) VALUES ($1, $2, $3) RETURNING id, handle, email, password;', [handle, password, email,])
```

Comment:
maybe shorten this line? put the sql string as a variable on the previous line, and pass the variable into `exec_params`?